scripts/filter_loop.py

```
@@ -117,8 +130,9 @@ def main() -> None:
         stamp = time.strftime("%Y-%m-%d %H:%M:%S")
         t0 = time.time()
         try:
+            top_n = _get_top_markets()
             r = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)